sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuWindowExec.scala

```diff
@@ -217,6 +225,7 @@ class GpuWindowExecMeta(windowExec: WindowExec,
     } catch {
       case _: NoSuchMethodException =>
         resultColumnsOnly = true
+        val methodNames = windowExec.getClass.getMethods().map(_.getName).toList
```

Comment:
Where is `methodNames` used?

Reply:
oops, old debug code, that should be removed.